src/comparisonOperators.js

```diff
@@ -0,0 +1,3 @@
+const comparisonOperators = ['=', '!=', '<', '>'];
```

Comment:
Does this file `comparisonOperators.js` do anything else apart from having a list of operators? I would rename this to say `utils.js` and have it contain all the utilities, like constants and perhaps utility functions